### DISCUSSION TITLE
Can't Edit autograding tests for some assignments

### DISCUSSION THREAD

**Discussion by schrum2** [community/community#80955] [GitHub Education]
### Hi everyone,

In one of my GitHub Classrooms I've been adding autograding tests to the assignments. However, after the initial creation of the assignment, some of my repositories are not allowing me to edit the tests. I click on the Edit Test button (looks like a pencil) and nothing happens. However, for other assignments in that same classroom, clicking the Edit button causes the sidebar to pop up immediately. However, I would say that for most, it does not work. At first I thought it might be an issue related to whether the assignment was a group or individual assignment, but that is not the case.

Incidentally, there are also some assignments that I can't even edit, because I get an error 500 Internal Server Error. These were created with the new Reuse Assignment Feature, so in the case of these assignments I've just been deleting the assignment and creating a new one.

**Comment by marknelsonengineer** [comment]
I'm having the exact same problem... I've used both Safari and Chrome.  The edit test button is definitely broke right now.

**Comment by irawoodring** [comment]
Same.  Huge pain to have to recreate them each time.

**Comment by schrum2** [comment]
This is still not working, and I would really like to be able to edit or at least have access to the commands I have used in an existing autograding test, because I need to retrieve some details from the command I used that are not available elsewhere.

**Reply by marknelsonengineer** [reply]
I know.  Me too.  It's been broken for four weeks now and GitHub hasn't addressed the issue yet.  

I can edit old autograder rules, but not new ones.

**Comment by alreadydea** [comment]
Response #2

**Comment by schrum2** [comment]
One helpful thing I just discovered:
If you check out a repo from the classroom, then you can find the details of how the autograding tests are configured by looking at the file:
```
.github\classroom\autograding.json
```
This can at least remind you of how the test was configured if you have to create a new but similar test.

Still, the Edit button in GitHub Classroom should work.

**Comment by mcknijak** [comment]
I've found the edit button doesn't work for me if the test is created as an "Run Command" test but if I create it as a "cpp" test, then it can be edited. Since you can use the "cpp" option to do the same things, I've found it useful as a workaround.

It's a feature they broke when they moved the test editor from a popout card window centered on the screen to one that shows up on the right side. Imo they shouldn't have implemented a cosmetic change if it broke actual functionality but I'm learning they don't bother to test any of their updates thoroughly...

**Comment by NachiketUN** [comment]
I have the same issue as @mcknijak, the edit button doesn't work for autograder tests. I face errors even when deleting the existing tests and creating new ones with the same name as the deleted ones.
